fsharp-backend/src/LibExecution/DvalReprLegacyExternal.fs

```diff
@@ -139,71 +133,12 @@ let ocamlStringOfFloat (f : float) : string =
 // Runtime Types
 // -------------------------
 
-// As of Wed Apr 21, 2021, this fn is only used for things that are shown to
-// developers, and not for storage or any other thing that needs to be kept
-// backwards-compatible.
-// CLEANUP: once we no longer support compatibility with OCaml, these messages can get much better.
-let rec typeToDeveloperReprV0 (t : DType) : string =
-  match t with
-  | TInt -> "Int"
-  | TFloat -> "Float"
-  | TBool -> "Bool"
-  | TNull -> "Null"
-  | TChar -> "Character"
-  | TStr -> "Str" // CLEANUP change to String
-  | TList _ -> "List"
-  | TDict _ -> "Dict"
-  | TRecord _ -> "Dict"
-  | TFn _ -> "Block"
-  | TVariable varname -> "Any"
-  | TIncomplete -> "Incomplete"
-  | TError -> "Error"
-  | THttpResponse _ -> "Response"
-  | TDB _ -> "Datastore"
-  | TDate -> "Date" // CLEANUP Dates should be DateTimes
-  | TPassword -> "Password"
-  | TUuid -> "UUID"
-  | TOption _ -> "Option"
-  | TErrorRail -> "ErrorRail"
-  | TResult _ -> "Result"
-  | TUserType (name, _) -> name
-  | TBytes -> "Bytes"
-
-let prettyTypename (dv : Dval) : string = dv |> Dval.toType |> typeToDeveloperReprV0
-
-// Backwards compatible version of `typeToDeveloperRepr`, should not be visible to
-// users (except through LibDarkInternal) but used by things like HttpClient
-// (transitively)
-let rec typeToBCTypeName (t : DType) : string =
-  match t with
-  | TInt -> "int"
-  | TFloat -> "float"
-  | TBool -> "bool"
-  | TNull -> "null"
-  | TChar -> "character"
-  | TStr -> "string"
-  | TList _ -> "list"
-  | TDict _ -> "dict"
-  | TRecord _ -> "dict"
-  | TFn _ -> "block"
-  | TVariable varname -> "any"
-  | TIncomplete -> "incomplete"
-  | TError -> "error"
-  | THttpResponse _ -> "response"
-  | TDB _ -> "datastore"
-  | TDate -> "date"
-  | TPassword -> "password"
-  | TUuid -> "uuid"
-  | TOption _ -> "option"
-  | TErrorRail -> "errorrail"
-  | TResult _ -> "result"
-  | TUserType (name, _) -> String.toLowercase name
-  | TBytes -> "bytes"
-
-// When printing to grand-users (our users' users) using text/plain, print a
-// human-readable format. TODO: this should probably be part of the functions
-// generating the responses. Redacts passwords.
-
+// SERIALIZER_DEF DvalReprLegacyExternal.toEnduserReadableTextV0
+// Plan: We'd like to deprecate this in favor of an improved version only
+// usable/used by StdLib functions in various http clients and middlewares.
+/// When printing to grand-users (our users' users) using text/plain, print a
+/// human-readable format. TODO: this should probably be part of the functions
```

Comment:
> TODO: this should probably be part of the functions
/// generating the responses.

I don't know what that says

Reply:
Nor do I - removed it.